public/offline.html

```diff
@@ -9,7 +9,7 @@
   <meta property="og:site_name" content="<%= community_qualified_name %>" />
 
   <meta name="twitter:card" content="summary_large_image">
-  <meta name="twitter:site" content="@ThePracticalDev">
+  <meta name="twitter:site" content="@<%= SiteConfig.social_media_handles["twitter"] %>">
```

Comment:
I don't think erb would work for this public page right?

Reply:
That is correct! Thanks for picking this up, I'm going to revert this change now for now and keep the hardcoded. I'll either figure out how to make the info on the offline page dynamic or we may just override it with a page from the backend as its already so specific. Will give it some more thought.